scripts/servers_test.py

```diff
@@ -1011,11 +1012,26 @@ def test_managed_webdriverio(self) -> None:
         self.assertIn('--suite full', program_args)
         self.assertIn('--params.devMode=True', program_args)
 
-    def test_managed_webdriverio_mobile(self) -> None:
+    @mock.patch('urllib.request.urlopen')
```

Comment:
Please use our `self.swap` or similar functions instead of `unittest.mock`. By convention, we use our own custom mocking tools instead of `unittest.mock` for oppia/oppia

Reply:
I have mocked with `self.swap`, please have a look. Thanks